docker-compose.yml

```diff
@@ -1,116 +1,124 @@
-# By default, this docker-compose setup uses Kafka 2.4.1. This version can
-# be overwritten by setting the KAFKA_IMAGE_TAG environment variable; some choices here include:
+# By default, this docker-compose setup uses Kafka 2.5.0
```

Comment:
This comment is inaccurate.

Reply:
Will fix it

Reply:
This has been fixed in the latest commit